cli/command/telemetry.go

```diff
@@ -122,7 +123,7 @@ func (r *telemetryResource) init() {
 	r.opts = nil
 }
 
-func (r *telemetryResource) defaultOptions() []resource.Option {
+func defaultOptions() []resource.Option {
```

Comment:
small tiny refactor here

Reply:
Yea this is fine. I sometimes like to attach functions to structs even when they don't use them just because Go lacks more proper namespaces. That way, I don't get confused about where the function is supposed to be invoked.

Reply:
If you make this a function instead of a method, maybe change the name to `defaultResourceOptions()` just to be more explicit.

Reply:
Also good for me, but i'd also rename the func as mentioned for that little bit of extra readability